automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoApiClient.scala

```diff
@@ -364,13 +363,13 @@ object LeonardoApiClient {
             method = Method.POST,
             headers = Headers(authHeader, defaultMediaType, traceIdHeader),
             uri = rootUri
-              .withPath(Uri.Path.unsafeFromString(s"/api/google/v1/disks/${googleProject.value}/${diskName.value}")),
+              .withPath(Uri.Path.unsafeFromString(s"/api/google/v1/disks/${googleProject.value}/${diskName}")),
```

Comment:
I don't think any of the changes from `${diskName.value}` to `diskName` are valid in this file. `AzureDiskName` and `DiskName` both seem to have the same `final case class A(value: String) extends AnyVal` definition, meaning usage in string interpolation like done here will result in a string `"...A(value)...".